lib/book.ex

```diff
@@ -0,0 +1,76 @@
+defmodule ExIsbndb.Book do
+  @moduledoc """
+  The `ExIsbndb.Book` module contains all the available endpoints for Books.
+
+  All functions need to receive a map with params, but only those
+  needed for the endpoint will be taken.
+  """
+
+  alias ExIsbndb.Client
+
+  @valid_column_values ["", "title", "author", "date_published"]
+
+  @doc """
+  Returns an instance of Book
+
+  ## Examples
+
+      iex> ExIsbndb.Book.get(1234567789)
+      {:ok, %Finch.Response{body: "...", headers: [...], status: 200}}
+
+  """
+  @spec get(map()) :: {:ok, Finch.Response.t()} | {:error, Exception.t()}
+  def get(isbn13) when is_binary(isbn13) do
+    Client.request(:get, "book/#{isbn13}", %{})
```

Comment:
```suggestion
    Client.request(:get, "book/#{isbn13}")
```

No need to add the empty map because `Client.request/3` defaults it 👍